confidential-data-hub/hub/src/hub.rs

```diff
@@ -5,11 +5,15 @@
 
 use async_trait::async_trait;
 use base64::{engine::general_purpose::STANDARD, Engine};
+use kms::{Annotations, ProviderSettings};
 use secret::secret::Secret;
 
 use crate::{DataHub, Error, Result};
 
-pub struct Hub {}
+pub struct Hub {
+    /// the get resource provider type. Semantically same as kbc.
+    get_resource_provider: String,
```

Comment:
Hi @fitzthum . The underlying driver to support the `get_resource` is exactly the kbcs in our current AA. I'm wondering if `sev-*-kbc` are still used now? If not we do not need this field to distinguish whether `kbs-client` (cc-kbc) or `sev-*-kbc` is used to be responsible for this.

If yes, we will need this parameter to specify. This parameter should be delivered from kata-agent from kernelcmdline, which we will use a workaround for this.